### DISCUSSION TITLE
My home consumption

### DISCUSSION THREAD

**Discussion by Monacoslo** [WillCodeForCats/solaredge-modbus-multi#621] [Q&A] [answered]
Where can I see my current home total consumption? I can see it in the SolarEdge app, but here I can not find it

**Comment by WillCodeForCats** [comment]
If you don't have a consumption meter then you will have to calculate the value from the Export+Import meter and inverter production. The solaredge monitoring platform does calculations, but the modbus integration only has data from the inverter (no calculations).

**Comment by Monacoslo** [comment]
I have consumption meter connected to SolarEdge inverter, that's how I get the correct info in Solar Edge mobile app. I just can not find the corresponding data in this integration output (or maybe do not know how to look).

**Reply by Monacoslo** [reply]
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/bd3c0d43-67a5-4ac4-984b-8643f4007d4a)
I can even see these 2 devices

**Reply by Monacoslo** [reply]
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/c4f41f1a-dfd3-4416-a1db-a42459b1ea77)
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/bfeb6305-a8ef-41af-b485-12a619c40bcb)
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/666700f5-90bb-4d16-9368-8537b2fe0ffb)
Maybe I just don't know which entity is what?

**Reply by Monacoslo** [reply]
AC power on SE16 is production?
What is then consumption and what is export?
AC power on M1 is export? What is consumption?

**Reply by Monacoslo** [reply]
I can see consumption within official integration, but this is via cloud and with huge delay
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/d0b0bf4c-4e1c-4c70-93cd-b83e9bc3fe58)

**Comment by Monacoslo** [comment]
Since I am targeting maximum allowed consumption power in my home, I need accurate realtime data.

**Reply by mpredfearn** [reply, marked as answer]
@Monacoslo use these template sensors https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Sensors-for-Power-and-Energy#single-inverter-single-import--export-meter

**Reply by Monacoslo** [reply]
> @Monacoslo use these template sensors https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Sensors-for-Power-and-Energy#single-inverter-single-import--export-meter

I have weird issue in my HomeAssistant. I copied templates to my templates.yams, but sensors were unavailable. Then I created template helpers via UI and I get the following error:
Sensor None has device class 'power', state class 'measurement' unit 'W' and suggested precision 'None' thus indicating it has a numeric value; however, it has the non-numeric value: '"601.0" availability: "True"' (<class 'str'>)

And value is shown as unknown, but graph shows values:
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/481ebb3c-1d02-4c9b-992d-7eb44c1bd16b)
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/7ef8466c-5642-4593-a6be-c87bef12cc64)

**Comment by Monacoslo** [comment]
I created them again using template yaml. But only one si showing values
![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/47989822/304b8c9d-9187-4226-b870-0c58d467b9e7)
Other have status "Unavailable", but individual sensors in code have values
